.github/workflows/e2e_lighthouse_performance_acceptance_tests.yml

```diff
@@ -122,79 +72,32 @@ jobs:
-            ${{ runner.os }}-build-
-            ${{ runner.os }}-
-      - name: Attempt to download build files artifact
-        id: download_artifact_1
-        uses: actions/download-artifact@v3
-        continue-on-error: true
-        with:
-          name: build_files
-          path: /home/runner/work/oppia/
-      - name: Unzip build files
-        if: steps.download_artifact_1.outcome != 'failure'
+      - name: Free disk space
         run: |
-          ls -la
-          unzip build_files.zip
-          rm build_files.zip
-          echo "Files in ./:"
-          ls -la .
-          echo "Files in oppia_tools:"
-          ls -la oppia_tools
-          echo "Files in oppia:"
-          ls -la oppia
-          echo "Files in build:"
-          ls -la oppia/build
-          echo "Files in third_party:"
-          ls -la oppia/third_party
-        working-directory: /home/runner/work/oppia
-        shell: bash
-      - uses: ./.github/actions/install-oppia-dependencies
-      - name: Install chrome
-        run: python -m scripts.install_chrome_for_ci
+          sudo rm -rf /usr/share/dotnet
+          sudo rm -rf "$AGENT_TOOLSDIRECTORY" 
+      - uses: ./.github/actions/install-chrome
       - name: Install ffmpeg
         run: sudo apt install ffmpeg
-      - name: (Artifact re-build) Cache third_party/static
-        if: steps.download_artifact_1.outcome == 'failure'
-        uses: actions/cache@v3
-        env:
-          cache-name: cache-node-modules
-        with:
-          path: /home/runner/work/oppia/oppia/third_party/static
-          key: ${{ runner.os }}-build-${{ env.cache-name }}-${{ hashFiles('dependencies.json') }}
-          restore-keys: |
-            ${{ runner.os }}-build-${{ env.cache-name }}-
-            ${{ runner.os }}-build-
-            ${{ runner.os }}-
-      - name: (Artifact re-build) Build Webpack
-        if: steps.download_artifact_1.outcome == 'failure'
-        run: python -m scripts.build --prod_env
+      - name: Initializing Containers
+        run: |
+          make build
+          # This is required as E2E tests are run on the host machine and not in the docker container.
+          docker compose up angular-build -d
+          sudo docker cp oppia-angular-build:/app/oppia/node_modules .
       - name: Run E2E Test ${{ matrix.suite.name }}
         uses: oppia/retry@a9fb265410e634c84ee90f3e87f323fde0541037
         with:
           max_attempts: 2
           substrings_indicating_flaky_execution: ${{ matrix.suite.flaky_indicators || '' }}
           command: >
-            VIDEO_RECORDING_IS_ENABLED=0;
-            xvfb-run -a --server-args="-screen 0, 1285x1000x24"
-            python -m scripts.run_e2e_tests --skip-install
-            --skip-build --suite=${{ matrix.suite.name }} --prod_env
+            sudo xvfb-run -a --server-args="-screen 0, 1285x1000x24" make run_tests.e2e suite=${{ matrix.suite.name }} VIDEO_RECORDING_IS_ENABLED=1
```

Comment:
Unless you know that video recordings don't increase flakiness, please turn them off by default

Reply:
Turned off

Reply:
I still see `VIDEO_RECORDING_IS_ENABLED=1`, which seems to indicate that video recordings are still enabled

Reply:
Fixed.